Marlin/Marlin_main.cpp

```diff
@@ -5503,10 +5511,13 @@ inline void gcode_M226() {
    * M300: Play beep sound S<frequency Hz> P<duration ms>
    */
   inline void gcode_M300() {
-    uint16_t beepS = code_seen('S') ? code_value_short() : 110;
-    uint32_t beepP = code_seen('P') ? code_value_long() : 1000;
-    if (beepP > 5000) beepP = 5000; // limit to 5 seconds
-    buzz(beepP, beepS);
+    uint16_t const frequency = code_seen('S') ? code_value_short() : 110;
```

Comment:
What is the effect of `const` here? I would expect it to produce a compiler error because `const` needs to be assigned at compile-time. Or is that only `static const` inside a method?